### PR TITLE
Added @reshadow/prettier

### PR DESCRIPTION
**Pull request by comerc**
- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to the `CHANGELOG.unreleased.md` file following the template.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by duailibe** [comment]
@comerc I think you need to run Prettier on the changed file

File: docs/plugins.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ Providing at least one path to `--plugin-search-dir`/`pluginSearchDirs` turns of
 - [`prettier-plugin-solidity`](https://github.com/prettier-solidity/prettier-plugin-solidity) by [**@mattiaerre**](https://github.com/mattiaerre)
 - [`prettier-plugin-svelte`](https://github.com/UnwrittenFun/prettier-plugin-svelte) by [**@UnwrittenFun**](https://github.com/UnwrittenFun)
 - [`prettier-plugin-toml`](https://github.com/bd82/toml-tools/tree/master/packages/prettier-plugin-toml) by [**@bd82**](https://github.com/bd82)
+- [`@reshadow/prettier`](https://github.com/lttb/reshadow/tree/master/packages/prettier) by [**@lttb**]
```
Comment (j-f1) [inline docs/plugins.md]:
```suggestion
- [`@reshadow/prettier`](https://github.com/lttb/reshadow/tree/master/packages/prettier) by [**@lttb**](https://github.com/lttb)
```

Comment (lipis) [inline docs/plugins.md]:
```suggestion
- [`@reshadow/prettier`](https://github.com/lttb/reshadow/tree/master/packages/prettier) by [**@lttb**]
```

**Comment by lipis** [comment]
Re `reshadow` is in 0.0.1-alpha.73.. maybe we could wait until something more stable.

**Pull request closed without merging**